tests/unit/stream_alert_rule_processor/test_rule_helpers.py

```diff
@@ -73,3 +73,53 @@ def test_in_network():
 
     ip_not_in_cidr = '10.0.15.24'
     assert_equal(base.in_network(ip_not_in_cidr, cidrs), False)
+
+def test_fetch_values_by_datatype():
+    """Helpers - Fetch values from a record by normalized type"""
+    rec = {
```

Comment:
Can we remove the unicode (`u'..`) prefix from these strings, or are they required for your tests? They typically aren't necessary unless you explicitly need the strings as unicode. Please also remove from your test on line 125 below if you can.